contracts/hydro/src/migration/migrate.rs

```diff
@@ -27,6 +26,10 @@ pub fn migrate(
         )));
     }
 
+    if contract_version.version == CONTRACT_VERSION_V2_0_4 {
```

Comment:
@p-offtermatt Please double-check the migration related stuff. We will be on v2.0.4 on staging/mainnet contracts at the moment this migration is run.